state/open.go

```diff
@@ -360,6 +369,28 @@ func (st *State) modelSetupOps(args ModelArgs, controllerInheritedConfig map[str
 	return ops, nil
 }
 
+func (st *State) createDefaultStoragePoolsOps(registry storage.ProviderRegistry) ([]txn.Op, error) {
+	m := poolmanager.MemSettings{make(map[string]map[string]interface{})}
```

Comment:
We have in state package already:
type attrValues map[string]interface{}

Could we use this type alias here too to make it a bit more readable?
m := poolmanager.MemSettings{make(map[string]attrValues)}